base/MAPL_EsmfRegridder.F90

```diff
@@ -1457,7 +1458,7 @@ subroutine create_route_handle(this, kind, rc)
      spec = this%get_spec()
 
      if (route_handles%count(spec) == 0) then  ! new route_handle
-        file_weights = IAND(spec%hints,REGRID_HINT_FILE_WEIGHTS) /= 0
+        file_weights = .true. !IAND(spec%hints,REGRID_HINT_FILE_WEIGHTS) /= 0
```

Comment:
@bena-nasa Do you know what this would do?

Reply:
We 100% do not want this @mathomp4